file_document_import_data/file_document.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#   account_statement_file_document for OpenERP
+#   Copyright (C) 2016-TODAY Akretion <http://www.akretion.com>.
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as
+#   published by the Free Software Foundation, either version 3 of the
+#   License, or (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import fields, orm
+import base64
+
+
+class FileDocument(orm.Model):
+    _inherit = "file.document"
+
+    def get_file_document_type(self, cr, uid, context=None):
+        res = super(FileDocument, self).get_file_document_type(
+            cr, uid, context=context)
+        res.append(('import_data', 'Import Data'))
+        return res
+
+    _columns = {
+        'import_model': fields.char('Model')
```

Comment:
On ir.model!
I have hesitated about it. But for the import, we just need the string, and I think that this kind of configuration is always made by odoo technical expert. So it is not probable that a mistake is made on this.
Also, if there is a mistake, it will be corrected very easily.
In my opinion, for a technical person it is easier to put the technical name of a model instead of choose in a big list of name.
For instance, easier to put purchase.order.line, than to search between 1000 models "Ligne de commande d'achat". 
Well this example is kind of easy, but some times, I don't even know the functional name of a model but I know the technical name.

Anyway, I am just explaining my position, but I don't mind changing!
What do you think?